statsmodels/discrete/discrete_model.py

```diff
@@ -3620,8 +3621,15 @@ def get_margeff(self, at='overall', method='dydx', atexog=None,
 
         Notes
         -----
+        Interpretations of methods:
+        - 'dydx' - change in `endog` for a change in `exog`
+        - 'eyex' - proportional change in `endog` for a proportional
+          change in `exog`
+        - 'dyex' - proportional change in `endog` for a change in `exog`
```

Comment:
I think these two are backward.  the `e` variable should be proportion, while the `d` variable is just change, so that dyex is change in y for proportional change in x.

Reply:
Two being this one and the next.

Reply:
Done

Reply:
it is my first ever PR so, next PRs will be cleaner :)